src/eventHandlers/functionEnvironmentReloadRequest.ts

```diff
@@ -34,6 +34,7 @@ export function functionEnvironmentReloadRequest(
                 logCategory: LogCategory.System,
             });
             process.chdir(msg.functionAppDirectory);
+            await channel.updatePackageJson('.');

```

Comment:
My tests revealed a bug in my earlier logic here. My ealier code was

```TS
await channel.updatePackageJson(msg.functionAppDirectory)
```

But because we're already `chdir`ing into the new `functionAppDirectory` in the line above, we would actually end up looking for `package.json` in `functionAppDirectory/functionAppDirectory/package.json`.

Need a second pair of eyes to validate my new logic here and if this makes sense.

Reply:
The path passed as `functionAppDirectory` should always be an absolute path (`/Users/eric/appDir` as opposed to `./appDir`), so it should work regardless of the current process's directory. My preference is to revert to your previous logic so that the order of these lines doesn't matter.

I'm guessing it's just the tests that were doing a relative path

Reply:
Yeah makes sense. I returned this code to the original and changed the tests to use absolute paths too. Thanks!